package-lock.json

```diff
@@ -51,6 +51,38 @@
         "react": "^16.14 || ^17 || ^18"
       }
     },
+    "../search-headless": {
```

Comment:
I think this is the problem, this is a relative path

Reply:
looks like it's fixed now!